app/controllers/katello/api/v2/repositories_controller.rb

```diff
@@ -181,6 +181,7 @@ def show
     param :id, :identifier, :required => true, :desc => N_("repository ID")
     param :source_url, String, :desc => N_("temporarily override feed URL for sync"), :required => false
     param :incremental, :bool, :desc => N_("perform an incremental import"), :required => false
+    param :force_full, :bool, :desc => N_("Force sync even if no upstream changes are detected"), :required => false
```

Comment:
We don't use the term 'full' in our descriptions so I wonder if we need it in the parameter name. `force` or `full_sync` would seem to cover it depending on how we want to position the parameter. With 'force' seeming to be closer given we are pushing it to do behaviors that are not natural.